app/helpers/icon_helper.rb

```diff
@@ -9,10 +9,6 @@ def icon(icon_classes:, classes: nil, **)
     content_tag(:i, nil, class: all_classes, **)
   end
 
-  def right_arrow_icon(**)
-    icon(icon_classes: 'bi bi-arrow-right', **)
-  end
```

Comment:
Dead code.

Reply:
Not anymore: https://github.com/sul-dlss/hungry-hungry-hippo/pull/878/files#diff-3e3bacebbd8b846ea714d0285efdd51b7a909b8690c3e3265f3714a1f2e1281bR2

Reply:
@justinlittman Thanks!